csvcubed/csvcubed/models/cube/qb/catalog.py

```diff
@@ -21,7 +21,7 @@ class CatalogMetadata(CatalogMetadataBase, UriIdentifiable):
     description: Optional[str] = field(default=None, repr=False)
     creator_uri: Optional[str] = field(default=None, repr=False)
     publisher_uri: Optional[str] = field(default=None, repr=False)
-    landing_page_uri: Optional[str] = field(default=None, repr=False)
+    landing_page_uri: Union[str, list[str], None] = field(default_factory=list, repr=False) # todo: fix type on left hand side
```

Comment:
I'd be tempted to rename this to `landing_page_uris` and leave the type as just `list[str]`. We don't need to support multiple different shapes here, a list will cover providing no information `[]`, a single value `['http://some-uri']` or multiple values `['http://first-uri', 'http://second-uri']`.